spec.html

```diff
@@ -111,10 +111,10 @@ <h1>
 			1. Let _callerContext_ be the running execution context.
 			1. Let _calleeContext_ be PrepareForWrappedFunctionCall(_F_).
 			1. Assert: _calleeContext_ is now the running execution context.
-			1. Let result be OrdinaryWrappedFunctionCall(_F_, _thisArgument_, _argumentsList_).
+			1. Let _result_ be Completion(OrdinaryWrappedFunctionCall(_F_, _thisArgument_, _argumentsList_)).
 			1. Remove _calleeContext_ from the execution context stack and restore _callerContext_ as the running execution context.
-			1. If result.[[Type]] is return, return result.[[Value]].
-			1. ReturnIfAbrupt(result).
+			1. ReturnIfAbrupt(_result_).
+			1. If _result_.[[Type]] is |return|, return _result_.[[Value]].
```

Comment:
Unfortunately I don't see how this fixes the issue:

- As stated in the original issue, `OrdinaryWrappedFunctionCall` can't produce a completion of `[[Type]]` _return_ AFAICT, wrapping it in a `Completion()` call doesn't change that fact, only makes it more explicit - https://tc39.es/ecma262/#sec-completion-ao:
 > The abstract operation Completion takes argument completionRecord (a [Completion Record](https://tc39.es/ecma262/#sec-completion-record-specification-type)) and returns a [Completion Record](https://tc39.es/ecma262/#sec-completion-record-specification-type). It is used to emphasize that a [Completion Record](https://tc39.es/ecma262/#sec-completion-record-specification-type) is being returned.
- `ReturnIfAbrupt` returns if the completion type is anything other than _normal_, which includes _return_, so the next line is unreachable